src/Styles/app.css

```diff
@@ -0,0 +1,50 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.nav-bar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-start;
+  padding: 1rem;
+  margin-bottom: 15rem;
+}
+
+.links {
+  margin-top: 1.9rem;
+  text-decoration: none;
+  font-size: 1rem;
+  font-weight: bold;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
```

Comment:
- Hey, great job! Please, let's kindly make some adjustments to our styles in the navbar so that it's in a particular position across all routes and doesn't have to be moving when changing the route as this behavior makes your design not ideal for a modern webpage. See the images below to address the issue I pointed out in red color below:
- Kindly add some more spaces in the links as they are too close and do not match that in the wireframe.


![Screenshot from 2023-01-19 21-58-22](https://user-images.githubusercontent.com/59397159/213559367-1f743955-67dc-4945-9d80-5b6be529adbb.png)
![Screenshot from 2023-01-19 21-56-51](https://user-images.githubusercontent.com/59397159/213559376-7c709a18-bbd9-4328-8a95-8c94098d9c30.png)
![Screenshot from 2023-01-19 21-55-12](https://user-images.githubusercontent.com/59397159/213559377-73062415-2554-4c2a-b663-b6867c23ffef.png)